.private/ci-build.sh

```diff
@@ -82,10 +82,12 @@ make -j4 -k
 if [ "${test}" = "yes" ]; then
 	# Load custom shim for WebUSB tests that simulates Web environment.
 	export NODE_OPTIONS="--require ${scriptdir}/../tests/webusb-test-shim/"
-	for test_name in init_context set_option stress stress_mt; do
+	for test_name in init_context macos set_option stress stress_mt; do
+	    if test -x ./tests/${test_name} ; then
```

Comment:
Why is this change necessary? It doesn't seem great that it might quietly skip some tests.

Reply:
Ok, opening a PR to add `make check`.

Reply:
See #1361

Reply:
This is much better, thanks.

Reply:
Merged into the PR. Done.